cmd/version.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/rancher/rke/cluster"
+	"github.com/urfave/cli"
+)
+
+func VersionCommand() cli.Command {
+	versionFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "cluster-file",
+			Usage:  "Specify an alternate cluster YAML file",
+			Value:  cluster.DefaultClusterConfig,
+			EnvVar: "CLUSTER_FILE",
+		},
+	}
+	return cli.Command{
+		Name:   "version",
+		Usage:  "Show Cluster Kubernetes version",
```

Comment:
In other commands descriptions, we have `cluster` lowercased, should probably do the same here